elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/CountryViewNested.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.example;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.datastores.aggregation.annotation.FriendlyName;
+
+import lombok.Data;
+
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * A nested view for testing.
+ */
+@Data
+@Include
+@Table(name = "countries")
+public class CountryViewNested {
```

Comment:
I would prefer if we created a model that reflects something meaningful.  It will be easier to understand the tests.

Reply:
When I work on Sashruts' PR I can change this view to ContinentView

Reply:
:+1: